earthpy/clip.py

```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
 
 
 def _clip_line_poly(shp, clip_obj):
```

Comment:
I'd use the same language that you used above for points for the lines function!